EVENT.md

```diff
@@ -0,0 +1,878 @@
+server side aggregation to provide richer data, which we are continuing to
+explore.
+
+## Related work
+
+There is an alternative [Private Click
+Measurement](https://github.com/privacycg/private-click-measurement) draft spec
+in the PrivacyCG. See this [WebKit blog
+post](https://webkit.org/blog/8943/privacy-preserving-ad-click-attribution-for-the-web/)
+for more details.
+
+There are multiple aggregate designs in
+[wicg/privacy-preserving-ads](https://github.com/WICG/privacy-preserving-ads),
+including Bucketization and MaskedLARK.
+
+Folks from Meta and Mozilla have published the [Interoperable Private
+Attribution(IPA)](https://docs.google.com/document/d/1KpdSKD8-Rn0bWPTu4UtK54ks0yv2j22pA5SrAD9av4s/edit#heading=h.f4x9f0nqv28x)
+proposal for doing aggregate attribution measurement.
+
+Brave has published and implemented an [Ads Confirmation
+Protocol](https://github.com/brave/brave-browser/wiki/Security-and-privacy-model-for-ad-confirmations).
+
+## API Overview
+
+### Registering attribution sources
+
+Attribution sources are events which future triggers can be attributed to. There
+are two types of attribution sources, `navigation` sources and `event` sources.
+
+`navigation` sources are registered via clicks on anchor tags:
+```html
+<a href="https://advertiser.example/landing"
+   attributionsrc="https://adtech.example/attribution_source?my_ad_id=123">
+   click me
+</a>
+```
+or via calls to `window.open` that occur with [transient
+activation](https://html.spec.whatwg.org/multipage/interaction.html#transient-activation):
+```javascript
+window.open(
+  "https://advertiser.example/landing",
+  "_blank",
+  "attributionsrc=https://adtech.example/attribution_source?my_ad_id=123");
+```
+
+ `event` sources do not require any user interaction and are registered via
+ `<a>` or `<img>` tags with the new `attributionsrc` attribute too:
+```html
+<a href="https://advertiser.example/landing"
+   registerattributionsrc
+   attributionsrc="https://adtech.example/attribution_source?my_ad_id=123" >
+</a>
+<!-- TODO: is view registration via an `<a>` tag necessary?
+    It requires a new attribute. -->
+
+<img src="https://advertiser.example/pixel" 
+     attributionsrc="https://adtech.example/attribution_source?my_ad_id=123" />
+```
+or via a JavaScript API:
+```javascript
+window.attributionReporting.registerAttributionSource(
+  "https://adtech.example/attribution_source?my_ad_id=123");
+```
+
+Each of these mechanisms will cause the browser to initiate a `keepalive` fetch
+request to the URL indicated by `attributionsrc`. The request will include a new
+HTTP header `Attribution-Reporting-Source-Info` which will include whether the
+source was a `navigation` or `event` source.
+
+The response to this request will configure the API. The browser will expect
+data in a new JSON HTTP header `Attribution-Reporting-Register-Source` which
+configures the API:
+
+```json
+{
+  source_event_id: "12340873456",
```

Comment:
This section is rendering in red, we should wrap the keys in quotes so it parses as valid JSON (and/or consider just dropping the specified)